### DISCUSSION TITLE
UI Library recommendation?

### DISCUSSION THREAD

**Discussion by RayRemnant** [timlrx/tailwind-nextjs-starter-blog#841] [Q&A]
Hello,
Love the project, I'm looking forward to use mdx to integrate different components in my blog pages. It would be great to have a UI library that works with it. So far it seems that most of them work only on client-side code, or require additional configuration. 
Do you have any suggetion in this regard? Thank you.

**Comment by PxlSyl** [comment]
Hi! I would recommend to try shadcnui, or even V.0 from Vercel (based on shadcnui and tailwind), it's a no brainer!
